src/kOS/Serialization/SerializationMgr.cs

```diff
@@ -11,22 +12,19 @@ public class SerializationMgr : SafeSerializationMgr
     {
         private readonly SharedObjects sharedObjects;
 
+        static SerializationMgr() {
+            SafeSerializationMgr.AddAssembly(typeof(SerializationMgr).Assembly.FullName);
+        }
```

Comment:
(See elsewhere in the PR comments for the comment with the word "curly brace" in big letters.)